src/lxc/lxccontainer.c

```diff
@@ -4659,6 +4680,158 @@ static bool do_lxcapi_restore(struct lxc_container *c, char *directory, bool ver
 
 WRAP_API_2(bool, lxcapi_restore, char *, bool)
 
+static int do_lxcapi_mount(struct lxc_container *c,
+			const char *source, const char *target,
+			const char *filesystemtype, unsigned long mountflags,
+			const void *data, struct lxc_mount *mnt) {
+	int fd;
+	char *template, *suff, *path;
+	pid_t pid, init_pid;
+	size_t len;
+	struct stat sb;
+	int ret = -1;
+
+	if (!c || !c->lxc_conf) {
+		ERROR("Container or configuration is NULL");
+		goto out_err;
+	}
```

Comment:
Actually, there's no need to use a `goto` here since you don't need to cleanup a bunch of stuff. So you should just:

    return -EINVAL

here.